fe2-android/app/src/main/java/com/github/dedis/popstellar/repository/remote/Connection.java

```diff
@@ -39,7 +73,8 @@ public Observable<WebSocket.Event> observeConnectionEvents() {
   }
 
   public void close() {
-    disposable.dispose();
+    disposables.dispose();
+    messagesSubject.onComplete();
```

Comment:
please add a short comment for its purpose

Reply:
added